torq.q

```diff
@@ -490,6 +491,8 @@ loadf0:{[reload;x]
   if[not[reload]&x in loadedf;.lg.o[`fileload;"already loaded ",x];:()];
   .lg.o[`fileload;"loading ",x];
   // error trapped loading of file
+  if[.finspace.enabled & x like "/opt/kx/app/db/*";
```

Comment:
having a hardcoded path here doesn't sit right with me, do we think this is a temporary measure or will this be here going forward? If it's going to be kept, can this be based on an env variable or something?

or perhaps we could create a more generic "blocklist" of locations to skip when loading, that could be configured in the FinTorQ app or something (in `env.q` maybe)?